show/main.py

```diff
@@ -1384,7 +1384,12 @@ def runningconfiguration():
 def all(verbose):
     """Show full running configuration"""
     cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    output = json.loads(run_command(cmd, display_cmd=verbose, return_cmd=True))
```

Comment:
>display_cmd [](http://example.com/codeflow?start=41&length=11)

`run_command` will internally call `click.echo()` or `sys.exit()`, so there will be side-effect on corner cases.